doc/articles/wpf-migration.md

```diff
@@ -6,16 +6,16 @@ This article covers architecture and implementation considerations for migrating
 
 ## Introduction
 
-While improvements will continue to be made by Microsoft at the framework level to ensure the compatibility and reliability of your WPF application, it may be desirable to expand its reach. In this hybrid work era, previous assumptions and development decisions which served your customers well become less valid. For instance, applications targeting the Windows platform alone miss out on mobility benefits of the Web. 
+While improvements will continue to be made by Microsoft at the framework level to ensure the compatibility and reliability of your WPF application, it may be desirable to expand its reach. In this hybrid work era, previous assumptions and development decisions which served your customers well become less valid. For instance, applications targeting the Windows platform alone miss out on mobility benefits of the Web.
 
-Advancements such as [WebAssembly (WASM)](https://developer.mozilla.org/docs/WebAssembly), a low-level bytecode for the web, have already become a viable, performant way to deliver your modern .NET applications.
+Advancements such as [WebAssembly (WASM)](https://developer.mozilla.org/docs/WebAssembly), a low-level byte-code for the web, have already become a viable, performant way to deliver your modern .NET applications.
```

Comment:
`bytecode` is a commonly used spelling, we can also add it to the exclusions.

Reply:
✅ Done